tasks/destroy.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Clear storeStorageDirectory
+  lineinfile:
+    path: /etc/beegfs/beegfs-storage.conf
+    regexp: "^storeStorageDirectory"
+    line: "storeStorageDirectory        ="
+  become: yes
+
+- name: Unmount beegfs data
+  mount:
+    path: "{{ beegfs_path_oss }}/{{ item }}"
+    state: absent
+  become: yes
+  with_items:
+  - "{{ beegfs_dev_oss }}"
+  when: beegfs_enable.oss | bool
+
+- name: Format filesystem
+  filesystem:
+    dev: "{{ item }}"
+    fstype: "{{ beegfs_fstype }}"
+    force: yes
+  become: yes
+  with_items:
+  - "{{ beegfs_dev_oss }}"
+  when:
+  - beegfs_enable.oss | bool
+  - beegfs_dev_oss is defined
+
+- name: Remove all beegfs folders
+  file:
+    path: /data/beegfs/
```

Comment:
Is this path always correct?

Reply:
Perhaps better to iterate over the paths that we definitely know exist! This is only correct for the current configuration.